xblockutils/studio_editable_test.py

```diff
@@ -0,0 +1,54 @@
+from selenium.webdriver.support.ui import WebDriverWait
+from xblockutils.base_test import SeleniumXBlockTest
+
+
+class StudioEditableBaseTest(SeleniumXBlockTest):
```

Comment:
Good point, but if you follow the chain of inheritance, this is actually using bok-choy. https://github.com/edx/xblock-sdk/blob/master/workbench/test/selenium_test.py